### ISSUE TITLE
Audio in looping video element needs controls  to pause or stop audio that plays automatically and is longer than 3 seconds

### ISSUE THREAD

**Issue by Rafael-Luna97** [Rafael-Luna97/second-github-integration#3]
**Rule description**  
One or more video elements, without an ARIA-assigned role, not intentionally hidden in the DOM, were found to have a loop attribute.

**Tool name**  
Access Engine

**Severity**  
High

**WCAG Success criteria**  
[1.4.2 Audio Control - Level A](https://www.w3.org/WAI/WCAG22/Understanding/audio-control)

**Why it matters**  
When video contains audio that plays automatically and is longer than 3 seconds it may prevent users of text to speech or screen readers who are blind or low vision from hearing content announced on the page or it may be a distraction to users with cognitive or learning disabilities.

**What to do**  
Audio-only or audio that is part of multimedia needs to not play automatically when the page loads. If audio content must load automatically, ensure the audio output plays for a maximum of 3 seconds or there is a mechanism near the top of the page to stop or pause the audio. Typical mechanisms include a button to stop/pause audio or a link to jump to the audio player where controls are present to stop or pause audio.

Use of the <code>autoplay</code> with <code>loop</code> attribute should be avoided unless there is a method to pause/stop/mute the audio.</p>

**Task in the Level Access Platform**  
https://tenanta.localhost-dashboard.essentialaccessibility.com/my-tasks/6a7a30cf410cee05c8b1b588/6a87634350d8252d6ec4dbac

**Comment by Rafael-Luna97** [comment]
Task pushed to GitHub Issues by Rafael Luna from Level Access Platform.

Posted by Rafael Luna via Level Access

**Comment by Rafael-Luna97** [comment]
<a href="https://tenanta.localhost-dashboard.essentialaccessibility.com/tasks/6a87634350d8252d6ec4dbac/attachments/6a87634350d8252d6ec4dbaa/download" target="_blank" rel="noopener noreferrer">Linked Findings.csv</a>

**Comment by Rafael-Luna97** [comment]
Task pushed to GitHub Issues by Rafael Luna from Level Access Platform.

Posted by Rafael Luna via Level Access
